src/hooks/useGeographicalStateAndCountryFromRoute.ts

```diff
@@ -1,9 +1,12 @@
 import {useRoute} from '@react-navigation/native';
 import type {ParamListBase, RouteProp} from '@react-navigation/native';
 import {CONST as COMMON_CONST} from 'expensify-common/lib/CONST';
+import CONST from '@src/CONST';
 
 type CustomParamList = ParamListBase & Record<string, Record<string, string>>;
```

Comment:
What do you think to remove CustomParamList and instead use the type from [here](https://github.com/Expensify/App/pull/41970#discussion_r1598015819) (or similar)?

Reply:
I have changed it a bit, what about how it is done now?